atd-toolbox/isolate_vz_data_differences/docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+services:
+  import:
+    build: .
+    volumes:
+      - .:/app
+    stdin_open: true
+    tty: true
+    env_file:
+      - env
+    deploy:
+      resources:
+        limits:
+          memory: 4G
```

Comment:
We're mandating a ceiling on the memory this container can use. Bump this up if you can / want.